src/compat/libc/include/limits.h

```diff
@@ -118,4 +118,59 @@ Maximum number of bytes in a character, for any supported locale.
 //TODO life is pain
 #include <ctype.h>
 
+/* POSIX/SUS minimum requirements.
+ * These numbers come from SUS and are host independent. */
```

Comment:
Thanks,
btw is it required by picocom?

Reply:
They are. well, at least _POSIX_PATH_MAX is.